chart/prometheus-msteams/values.yaml

```diff
@@ -45,7 +45,22 @@ metrics:
     scrapeInterval: 30s
 
 connectors: []
-connectors_with_custom_templates: []
+
+# Templates defined here can be used in connectorsWithCustomTemplates and are mounted into /etc/template/
+customCardTemplates: []
+# - templateFileName: custom-message-card.tmpl
+#   content: |
+#     {{ define "teams.card" }}
+#     {...}
+#     {{ end }}
+
+# ref: https://github.com/prometheus-msteams/prometheus-msteams#customise-messages-per-ms-teams-channel
+# Templates are mounted into /etc/template/
+connectorsWithCustomTemplates: []
+# - request_path: /alert2
+#   template_file: /etc/template/custom-message-card.tmpl
+#   webhook_url: <webhook>
+#   escape_underscores: true
 
 ## Specify the custom message card template for MS teams
 # customCardTemplate: |
```

Comment:
see above, I think we can get rid of that.

Reply:
I'll remove it

Reply:
I'll remove it